charts/pega/templates/pega-environment-config.yaml

```diff
@@ -40,4 +40,6 @@ data:
   CASSANDRA_NODES: "{{ include "cassandraNodes" . }}"
   # Port to connect to cassandra with
   CASSANDRA_PORT: "{{ .Values.dds.port }}"
+  # Pass constellation URL as a Dynamic System Setting property of Pega Infinity.
+  NODE_SETTINGS: "PEGA-UIENGINE/CONSTELLATIONSVCURL=YOUR_DOMAIN_NAME_HERE:30080/prweb/constellation"
```

Comment:
Seems it should be constructed from helm values, but not hardcoded as soon as parts of this string exist in the values.